doc/user-guide/Push-notifications.md

```diff
@@ -1,4 +1,4 @@
-## Push Notifications
+## How to setup Push Notifications with MongoosePush
```

Comment:
"setup" or "set up"? :) `ICE_Tutorial.md` and `Push-notifications.md` are not consistent in this matter.